.github/workflows/auto-stale-pr.yml

```
@@ -0,0 +1,28 @@
+name: Mark stale pull requests
+
+on: pull_request
```

I am not sure on trigger, because it will be trigger on PR opening but then it's never stale.

On exemples I see that it's always a cron:
https://github.com/actions/stale?tab=readme-ov-file#usage

https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#schedule

```suggestion
on:
  schedule:
    - cron: '30 1 * * *'
```